amundsen_application/static/css/_layouts.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
+
```

Comment:
What are the changes in the file addressing? The metadata sections are no longer aligned in the way that we want them to be.

Reply:
Oh, I see, I will double check, thanks!

There was a huge space at the bottom of the sidebar that was triggering a y-scroll when it wasn't necessary.

Reply:
Oh, yeah, that was pretty obvious there, I don't know how it went through!

Just sent a fix.

Reply:
Taking another look, still have a few more comments. Essentially I would expect this PR to result in the page looking exactly the same except for what is needed to fix the issue you referenced here.